src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String welcomeMessage = "Hello! I'm KJ\nHow can I help you today?";
+
+    private static final String completeMessage = "Nice! I've marked this task as done:";
+
+    private static final String byeMessage = "Bye. Hope to see you again soon!";
+
+    private static final String listMessage = "Here are the tasks in your list:";
```

Comment:
Consider exporting magic strings out into a separate class file for easier management

Reply:
It does seem I have too many magic strings in this class, though I didn't think of exporting them into another class file. Thanks for the suggestion!